src/AppKit/Enums.cs

```diff
@@ -3009,4 +3013,48 @@ public enum NSWorkspaceAuthorizationType : long  {
 		SetAttributes,
 		ReplaceFile,
 	}
+
+	[Mac (10,16)]
+	[Native]
+	public enum NSTableViewStyle : long
+	{
+		Automatic,
+		FullWidth,
+		Inset,
+		SourceList
+	}
+
+	[Mac (10,16)]
+	[Native]
+	public enum NSTitlebarSeparatorStyle : long
+	{
+		Automatic,
+		None,
+		Line,
+		Shadow
+	}
+
+	[Mac (10,16)]
+	[Native]
+	public enum NSWindowToolbarStyle : long
+	{
+		Automatic,
+		Expanded,
+		Preference,
+		Unified,
+		UnifiedCompact
+	}
+
+	[Flags, Mac (10,16)]
+	[Native]
+	public enum NSTableViewAnimationOptions : ulong
+	{
+		EffectNone = 0x0,
+		EffectFade = 0x1,
+		EffectGap = 0x2,
+		SlideUp = 0x10,
+		SlideDown = 0x20,
+		SlideLeft = 0x30,
+		SlideRight = 0x40
```

Comment:
minor: add a `,` so any future addition won't make the last value part of the diff